app/src/main/java/com/marverenic/music/data/store/LocalMusicStore.java

```diff
@@ -219,11 +219,11 @@ private String getDirectoryInclusionExclusionSelection() {
         String excludeSelection = getDirectoryExclusionSelection();
 
         if (includeSelection != null && excludeSelection != null) {
-            selection = "(" + includeSelection + ") AND (" + excludeSelection + ")";
+            selection = "((" + includeSelection + ") AND (" + excludeSelection + "))";
         } else if (includeSelection != null) {
-            selection = includeSelection;
+            selection = "(" + includeSelection + ")";
         } else if (excludeSelection != null) {
-            selection = excludeSelection;
+            selection = "(" + excludeSelection + ")";
```

Comment:
Instead of adding all of the parenthesis here, we could just change the return statement to:

```java
return "(" + selection + ")";
```